tools/PI/DevHome.PI/Views/BarWindow.cs

```diff
@@ -26,7 +26,7 @@ public partial class BarWindow
     private readonly Settings _settings = Settings.Default;
     private readonly BarWindowHorizontal _horizontalWindow;
     private readonly BarWindowVertical _verticalWindow;
-    private readonly BarWindowViewModel _viewModel = Application.Current.GetService<BarWindowViewModel>();
+    private readonly BarWindowViewModel _viewModel = new();
```

Comment:
Agreed, it seems that instead of deleting
```services.AddSingleton<BarWindowViewModel>();```
it should have become
```services.AddTransient<BarWindowViewModel>();```

Reply:
I wonder if a factory is the "right" way to do this (not a big deal but good to have in your back pocket if this needs to be more DI-y in the future)